.github/workflows/spack_cpu_build.yaml

```
@@ -30,23 +98,51 @@ jobs:
     name: Build ExaGO with Spack
```

```suggestion
    name: Build
```

@jaelynlitz since we already have the `Spack Ubunutu x86_64 Buildcache` at the start of the pipeline name, this would give more of the spack spec in the pipeline name. We can also consider shortening that name